win32ss/user/ntuser/desktop.c

```diff
@@ -260,6 +261,17 @@ InitDesktopImpl(VOID)
                       SynchronizationEvent,
                       FALSE);
 
+    /* Check if user has disabled window snapping */
+    DWORD EntryValue = 0;
+    BOOL HasEntry = RegReadUserSetting(L"Control Panel\\Desktop",
+                                       L"WindowArrangementActive",
+                                       REG_SZ, &EntryValue, sizeof(EntryValue));
+    if (HasEntry)
```

Comment:
```suggestion
    if (RegReadUserSetting(L"Control Panel\\Desktop",
                           L"WindowArrangementActive",
                           REG_SZ, &EntryValue, sizeof(EntryValue)))
```

Reply:
Also, why isn't this done in SpiUpdatePerUserSystemParameters() ?

Reply:
I was told on IRC it would be more appropriate to do it here since we're not sure how Windows does it, I can move it there if it's more appropriate